recipe/thirdparty/providers/googleWorkspaces.go

```diff
@@ -1,167 +1,43 @@
-/* Copyright (c) 2021, VRAI Labs and/or its affiliates. All rights reserved.
- *
- * This software is licensed under the Apache License, Version 2.0 (the
- * "License") as published by the Apache Software Foundation.
- *
- * You may not use this file except in compliance with the License. You may
- * obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
- * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
- * License for the specific language governing permissions and limitations
- * under the License.
- */
-
 package providers
 
 import (
-	"errors"
-	"strings"
-
-	"github.com/golang-jwt/jwt/v4"
-	"github.com/supertokens/supertokens-golang/recipe/thirdparty/api"
 	"github.com/supertokens/supertokens-golang/recipe/thirdparty/tpmodels"
 	"github.com/supertokens/supertokens-golang/supertokens"
 )
 
-const googleWorkspacesID = "google-workspaces"
+func GoogleWorkspaces(input tpmodels.ProviderInput) tpmodels.TypeProvider {
+	input.ThirdPartyID = "google-workspaces"
```

Comment:
if the input id is not defined, only then set this

Reply:
Done